README.md

```diff
@@ -116,8 +116,23 @@ Click the button below to start deployment.
    - **App Client ID** - Type the GUID of your application that created the subscription. 
    
    **Note:** You can obtain this value from _Azure_ > _AD_ > _App registrations_ > _Your app name_
+
    - **App Client Secret** - Type the secret key of your application (available from from _App Registrations_).
 
+   **Note:** The following template parameters are optional and need to be entered only if reusing an existing Event Hub:
+
+   - **Event Hub Resource Group** - Type the resource group for the existing Event Hub.
+   - **Event Hub Connection String** - Type the connection string for the existing Event Hub.
```

Comment:
`for the existing Event Hub or Event Hub Namespace`.
This link may be helpfull https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-get-connection-string

Reply:
Leaving as is to not complicate the instructions.